lib/java/change-directory/src/test/java/org/enso/change/directory/TestChangeDirectory.java

```diff
@@ -0,0 +1,76 @@
+package org.enso.change.directory;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import org.graalvm.nativeimage.ImageInfo;
+import org.junit.Test;
+
+public class TestChangeDirectory {
+  public TestChangeDirectory() {}
+
+  private static void ensureInNativeImage() {
+    var inNativeImage = ImageInfo.inImageRuntimeCode();
+    assumeTrue(
+        "This test should only be run in a native image. Please run it in a native image.",
+        inNativeImage);
+  }
+
+  @Test
+  public void curDir() {
```

Comment:
The order of methods as executed by JUnit doesn't follow the order in the source code. Make sure the tests are "idempotent"  and don't influence each other.